test/resources/versioning/DockerArtifactVersioning/Dockerfile_registryPort

```diff
@@ -0,0 +1,5 @@
+FROM registry:4444/path/image:1.2.3
```

Comment:
do we need to support this: `registry:4444/path/image`

Reply:
yes, this is the typical path for a custom registry

Reply:
we already had one occurrence where this was relevant

Reply:
I mean without an explicit version for the image, I guess the code will fail for that.

Reply:
explicit error message added for such a case